src/utils/relativeTime.js

```diff
@@ -0,0 +1,80 @@
+import i18n from '@dhis2/d2-i18n'
+
+// our locales use _ instead of - for locales
+const language = i18n.language?.replace('_', '-') || 'en-gb'
```

Comment:
you probably also need to do something for locales with a script variant `uz_UZ_Cyrl`

Reply:
Do you know the format this would be returned as from the DHIS2 API?

Reply:
yeah this is the format returned `uz_UZ_Cyrl`